torch/_torch_docs.py

```diff
@@ -2210,7 +2210,7 @@ def merge_dicts(*dicts):
 
     >>> a = torch.randn(3, 3)
     >>> a = a @ a.mT + 1e-3 # make symmetric positive-definite
-    >>> l = torch.cholesky(a)
+    >>> l = torch.linalg.cholesky(a)
```

Comment:
This example calling cholesky shouldn't be changed to torch.linalg.cholesky because it's an example for the torch.cholesky operation

Reply:
You're right. Thank you for pointing out.